lib/course.rb

```diff
@@ -1 +1,22 @@
+class Course 
+  attr_reader :name
+  attr_accessor :capacity, 
+                :students
+                
 
+  def initialize(name, capacity)
+    @name = name
+    @capacity = capacity
+    @students = []
+    @full = false
+  end
+
+  def full?
+    @full
+  end
```

Comment:
You already have attributes available that will let you check this. I don't think we need `@full` as an attribute at all:
```suggestion
  def full?
    @students.count >= @capacity
  end
```

Reply:
Ahhhh, yes. The poor decisions made in the panic of the timed test.